### PR TITLE
Feature/timeseries chart

### PR DESCRIPTION
**Pull request by mekiert**

### CODE REVIEW

File: frontend/src/app/modules/time-series/services/line-chart.service.ts [hunk 1]
```diff
@@ -535,57 +692,70 @@ this.calculateLegendDimensions();
...
+  //           .attr('d', (dataItem: TimeSeries) => {
+  //             let newDataValues = dataItem.values.filter((point) => {
+  //               return point.date <= maxDate && point.date >= minDate;
+  //             });
+  //             return this.getLineGenerator(xScale, yScale)(newDataValues);
+  //           })
+  //       }
+  //     )
+  //   }
+  // }
```
Comment (BennyLi) [inline frontend/src/app/modules/time-series/services/line-chart.service.ts]: Remove unused code.

File: grails-app/i18n/messages.properties [hunk 2]
```diff
@@ -1277,6 +1277,20 @@ frontend.de.iteratec.osm.barchart.filter.noFilterAsc=Ascending
 frontend.de.iteratec.osm.barchart.filter.label=Filter
 frontend.de.iteratec.osm.barchart.filter.customerJourneyHeader=Customer Journey
 frontend.de.iteratec.osm.timeSeries.loadTimes=Load times
+frontend.de.iteratec.chart.contextMenu.summary=Summary
+frontend.de.iteratec.chart.contextMenu.waterfall=Waterfall
+frontend.de.iteratec.chart.contextMenu.performanceReview=Performance Review
+frontend.de.iteratec.chart.contextMenu.contentBreakdown=Content Breakdown
+frontend.de.iteratec.chart.contextMenu.domains=Domains
+frontend.de.iteratec.chart.contextMenu.screenshot=Screen Shot
```
Comment (BennyLi) [inline grails-app/i18n/messages.properties]: Please change it to `Screenshot`

**Comment by BennyLi** [comment]
Local testing looks good. Just one points from local testing:

- [ ] The mouse to data point association isn't as good as it was before (in the old graph). In the old graph the nearest dot was always on the line that was next to the mouse pointer. In the new graph it is the nearest point to the marker line, which makes it very difficult to get to some data points in the graph. Well, this is not related to this ticket. So maybe we should consider to open a bug issue for this.

**Review by BennyLi — CHANGES_REQUESTED** [review]

File: frontend/src/app/modules/time-series/services/line-chart.service.ts (hunk 1 above)
Comment (mekiert) [reply frontend/src/app/modules/time-series/services/line-chart.service.ts]: Done.

File: grails-app/i18n/messages.properties (hunk 2 above)
Comment (mekiert) [reply grails-app/i18n/messages.properties]: Done.

**Review by BennyLi — APPROVED** [review]

**Pull request merged**